.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   parserOptions: {
     project: 'tsconfig.json',
     sourceType: 'module',
+    extraFileExtensions: ['.json'],
```

Comment:
revert this. This error was arising due to json file `example.service.json` but it is fixed in the previous commit.